### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Eliott-Guevel**
The PharoStreamDeck plugin is complete, alongside its documentation in the different GitHub folders.

### CODE REVIEW

**Comment by Eliott-Guevel** [comment]
@labordep Could you approve this pull request please?

File: Plugins/PharoStreamDeck/PharoStreamDeck JavaScript part/Sources/com.thales.pharostreamdeck.sdPlugin/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html>
 
 <head>
-	<title>com.thales.yearSelectorPharo</title>
+	<title>com.thales.supportwizarddeck</title>
```
Comment (labordep) [inline Plugins/PharoStreamDeck/PharoStreamDeck JavaScript part/Sources/com.thales.pharostreamdeck.sdPlugin/index.html]: remove all "supportwizard" terms, use "testapplication"

File: Plugins/PharoStreamDeck/PharoStreamDeck JavaScript part/Sources/com.thales.pharostreamdeck.sdPlugin/manifest.json [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+      "Name": "YearSelector",
+      "States": [
+        {
+          "Image": "img/selectorIcon",
+          "TitleAlignment": "middle"
+        }
+      ],
+      "SupportedInMultiActions": false,
+      "Tooltip": "Permits the user to choose a value and highlights the last key pressed. This value is then displayed on a YearWitness",
+      "UUID": "com.thales.supportwizarddeck.yearselector",
```
Comment (labordep) [inline Plugins/PharoStreamDeck/PharoStreamDeck JavaScript part/Sources/com.thales.pharostreamdeck.sdPlugin/manifest.json]: "testapplication"

File: Plugins/PharoStreamDeck/PharoStreamDeck JavaScript part/Sources/com.thales.pharostreamdeck.sdPlugin/manifest.json [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+      "Name": "PercentSelector",
+      "States": [
+        {
+          "Image": "img/selectorIcon",
+          "TitleAlignment": "middle"
+        }
+      ],
+      "SupportedInMultiActions": false,
+      "Tooltip": "Permits the user to choose a value and highlights the last key pressed. This value is then displayed on a PercentWitness",
+      "UUID": "com.thales.supportwizarddeck.percentselector",
```
Comment (labordep) [inline Plugins/PharoStreamDeck/PharoStreamDeck JavaScript part/Sources/com.thales.pharostreamdeck.sdPlugin/manifest.json]: "testapplication"

File: Plugins/PharoStreamDeck/PharoStreamDeck JavaScript part/Sources/com.thales.pharostreamdeck.sdPlugin/manifest.json [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+        {
+          "Image": "img/actionIcon",
+          "TitleAlignment": "middle",
+          "TitleColor": "#c8c8c8",
+          "FontSize": "16"
+        }
+      ],
+      "SupportedInMultiActions": false,
+      "Tooltip": "Shows the year value, pressing it resets the value to 0",
+      "UUID": "com.thales.supportwizarddeck.yearwitness"
```
Comment (labordep) [inline Plugins/PharoStreamDeck/PharoStreamDeck JavaScript part/Sources/com.thales.pharostreamdeck.sdPlugin/manifest.json]: "testapplication"

File: Plugins/PharoStreamDeck/PharoStreamDeck JavaScript part/Sources/com.thales.pharostreamdeck.sdPlugin/manifest.json [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+      "States": [
+        {
+          "Image": "img/actionIcon",
+          "TitleAlignment": "middle",
+          "FontSize": "16"
+        }
+      ],
+      "SupportedInMultiActions": false,
+      "Tooltip": "Shows the percent (operational capacity) value, pressing it resets the value to 0",
+      "UUID": "com.thales.supportwizarddeck.percentwitness"
```
Comment (labordep) [inline Plugins/PharoStreamDeck/PharoStreamDeck JavaScript part/Sources/com.thales.pharostreamdeck.sdPlugin/manifest.json]: "testapplication"

File: Plugins/PharoStreamDeck/PharoStreamDeck JavaScript part/Sources/com.thales.pharostreamdeck.sdPlugin/manifest.json [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+      "States": [
+        {
+          "Image": "img/modelIcon",
+          "TitleAlignment": "middle",
+          "FontSize": "5"
+        }
+      ],
+      "SupportedInMultiActions": false,
+      "Tooltip": "Permits the user to choose a support model for the Support Wizard application",
+      "UUID": "com.thales.supportwizarddeck.model",
```
Comment (labordep) [inline Plugins/PharoStreamDeck/PharoStreamDeck JavaScript part/Sources/com.thales.pharostreamdeck.sdPlugin/manifest.json]: "testapplication"

File: Plugins/PharoStreamDeck/PharoStreamDeck JavaScript part/Sources/com.thales.pharostreamdeck.sdPlugin/manifest.json [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+      "Name": "Reset Configuration",
+      "States": [
+        {
+          "Image": "img/resetConfigurationIcon",
+          "TitleAlignment": "middle"
+        }
+      ],
+      "SupportedInMultiActions": false,
+      "Tooltip": "Permits the user to reset the configuration of the Support Wizard application",
+      "UUID": "com.thales.supportwizarddeck.resetconfiguration"
```
Comment (labordep) [inline Plugins/PharoStreamDeck/PharoStreamDeck JavaScript part/Sources/com.thales.pharostreamdeck.sdPlugin/manifest.json]: "testapplication"

**Review by labordep — CHANGES_REQUESTED** [review]
You should rewrite to change "supportwizard" to "testapplication".

**Comment by Eliott-Guevel** [comment]
Names were changed accordingly

**Pull request merged**